### ISSUE TITLE
WCAG2ICT Project: Discover if we need new introductory sections and add, if needed

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#20]
For other documents that get updated, there are sections describing changes between versions, background/history, etc.  Not sure if a Working Group Note typically has them, but these are worth considering:
- [x] Background on WCAG2ICT
- [x] Comparison with WCAG2ICT 2013 - add a placeholder section that will be written & reviewed prior to publication (tracked in #65)
- Change log - decision made on 10 Nov. to not incorporate a change log.

**Comment by maryjom** [comment]
Created a [Discussion poll](https://github.com/w3c/wcag2ict/discussions/48) for the group to weigh in on this topic.

**Comment by maryjom** [comment]
[Results of the poll](https://github.com/w3c/wcag2ict/discussions/48) are a bit mixed. 

- [Discussion of early results](https://www.w3.org/2022/11/03-wcag2ict-minutes#item01) in the 3 Nov. meeting during the Announcements
- Discussion during 10 Nov. meeting coming tomorrow.

**Comment by Lboniello** [comment]
A working group note for W3C on the topic of applying WCAG 2.0 to non-web information and communication technologies (WCAG2ICT) was approved in September of 2013. The note, [Guidance on Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict/) describes how the WCAG 2.0 can be applied to ICT, specifically to non-web documents and software.  The note is organized by sections to mirror those found in WCAG 2.0: perceivable, operable, understandable, and robust. It was intended that the note would help clarify when and how WCAG 2.0 success criteria should be applied: without modification or with modification (with edits and/or notes). Glossary terms were also reviewed for applicability as written. AAA was not addressed in the final note.

WCAG2ICT has been relied upon in regulations and standards used in regulations such as: [Section 508 - Application of WCAG 2.0 to non-web ICT (US)](https://www.access-board.gov/ict/#b-application-of-wcag-2-0-to-non-web-ict) and [EN 301 549 V3.2.1](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf#page=%5B13%5D) (Europe) and other standards that reference and/or incorporate EN 301 549 (e.g. India, Kenya, Australia, etc.). Section 508 looks to WCAG2ICT for detailed direction in providing specific guidance and exceptions to particular criteria from being applied to non web technology.

**Comment by bruce-usab** [comment]
In addition to USAB link (from the preamble to the final rule), the regulatory adoption of [WCAG Conformance](https://www.access-board.gov/ict/#E207.2) for non-web software and WCAG [Accessibility Standard](https://www.access-board.gov/ict/#E205.4) applied to non-web documents.

**Comment by ThorstenKatzmann** [comment]
editorial only, better "Level AAA Success Criteria" instead of only "AAA" so that it reads "Level AAA Success Criteria was not addressed in the final note."

**Comment by maryjom** [comment]
One minor edit to Thorsten's edit: "Level AAA Success Criteria **were** not addressed in the final note."

**Comment by maryjom** [comment]
@Lboniello Can you incorporate comments from Thorsten and Bruce into the draft for our discussion this Thursday?

**Comment by Lboniello** [comment]
@maryjom absolutely. Will comment with the updated draft in a clean comment window.

**Comment by Lboniello** [comment]
@bruce-usab - where did you want this?
> In addition to USAB link (from the preamble to the final rule), the regulatory adoption of [WCAG Conformance](https://www.access-board.gov/ict/#E207.2) for non-web software and WCAG [Accessibility Standard](https://www.access-board.gov/ict/#E205.4) applied to non-web documents.

**Comment by Lboniello** [comment]
Updated below without @bruce-usab addition as it was unclear to me how to incorporate that text/link/comment.

A working group note for W3C on the topic of applying WCAG 2.0 to non-web information and communication technologies (WCAG2ICT) was approved in September of 2013. The note, [Guidance on Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict/) describes how the WCAG 2.0 can be applied to ICT, specifically to non-web documents and software. The note is organized by sections to mirror those found in WCAG 2.0: perceivable, operable, understandable, and robust. It was intended that the note would help clarify when and how WCAG 2.0 success criteria should be applied: without modification or with modification (with edits and/or notes). Glossary terms were also reviewed for applicability as written. Level AAA Success Criteria were not addressed in the final note.

WCAG2ICT has been relied upon in regulations and standards such as: [Section 508 - Application of WCAG 2.0 to non-web ICT (US)](https://www.access-board.gov/ict/#b-application-of-wcag-2-0-to-non-web-ict) and [EN 301 549 V3.2.1](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf#page=%5B13%5D) (Europe) and other standards that reference and/or incorporate EN 301 549 (e.g. India, Kenya, Australia, etc.). Section 508 looks to WCAG2ICT for detailed direction in providing specific guidance and exceptions to particular criteria from being applied to non web technology.

**Comment by maryjom** [comment]
@Lboniello A clarification was asked for regarding Level AAA statement in the 1 Dec. meeting by Gregg Vanderheiden to be clear about it being the original note.  Now that I have thought about it more, maybe say "Level AAA Success Criteria were not addressed in the 2013 WCAG2ICT Note." 

I think once Bruce's comments are addressed then this can be incorporated into the GitHub document. Do you feel comfortable doing the GitHub pull request or wish to leave that to the editors to do?

**Comment by bruce-usab** [comment]
My apologies for missing the call toady, and for my comment above being so cryptic!

@Lboniello presently includes a link to the 508 *preamble* discussing the [Application of WCAG 2.0 to Non-Web ICT](https://www.access-board.gov/ict/#b-application-of-wcag-2-0-to-non-web-ict).  We all agree that is very helpful!

In _addition_ to referencing that preamble prose, my suggestion is to _also_ cite to the 508 regulation, showing the provisions that incorporate by reference the [WCAG Accessibility Standard as to be applied to non-web documents](https://www.access-board.gov/ict/#E205.4) and [WCAG Conformance for non-web software](https://www.access-board.gov/ict/#E207.2).

**Comment by bruce-usab** [comment]
@Lboniello (et al.) for linking to the preamble discussion, please use:  https://www.federalregister.gov/documents/2017/01/18/2017-00395/information-and-communication-technology-ict-standards-and-guidelines#h-36

(Linking to access-board.gov for 508 _provisions_ is the best option.  But it is not working well to have the preamble on the same page (on the USAB site) as the regulation, since printing results in about 100 more pages than people expect.  It is on my list of things to move the preamble its own page.)

**Comment by bruce-usab** [comment]
On call today (1/19) I volunteered to try my hand on a PR to resolve this thread.

**Comment by bruce-usab** [comment]
See proposed changes:  https://github.com/w3c/wcag2ict/pull/81/files

**Comment by pday1** [comment]
Minor editorial point: Guidance section, paragraph 1, line 2. 
Broken markdown link is still present "[[WCAG22]]" after the link Web Content Accessibility Guidelines (WCAG) 2.2

**Comment by maryjom** [comment]
@pday1 The broken link gets resolved when incorporated into the Editor's draft. There's special respec stuff that goes on when the full document is built to make that part work.

**Comment by maryjom** [comment]
AG WG has a [survey](https://www.w3.org/2002/09/wbs/55145/WCAG2ICT-First-AG-review-4-sc/) open until 9 March with AG WG survey results discussion at a TBD future date (after CSUN conference).

**Comment by maryjom** [comment]
All content complete for this section.
